### ISSUE TITLE
MobileTabItem can't include other view

### ISSUE THREAD

**Issue by j4587698** [kikipoulet/SukiUI#15]
I have a demo in https://github.com/j4587698/SukiUiTest

MainView only have a TabControl
```
  <TabControl Classes="MobileTabView">
      <TabItem Header="Test1" Classes="MobileTabItem">
          <views:TestView></views:TestView>
      </TabItem>
      <TabItem Header="Test2">
          <views:TestView></views:TestView>
      </TabItem>
  </TabControl>
```

TestView only have a string.

Run it
Test1 page is a empty page
![image](https://user-images.githubusercontent.com/24642446/211127833-98f165c1-f765-4dbf-a677-6f7100ec69b4.png)

Test2 page can show the string
![image](https://user-images.githubusercontent.com/24642446/211127846-8ebec5d0-838d-4a30-a63f-1e5f2f547a54.png)

**Comment by kikipoulet** [comment]
The text disappears because of the ScaleTransform effect for the header of the TabItem that is applied (should not) on the Content too when the Tab is selected. Your TestView is in fact scaled *1.4 :(

So I will make a new nuget package monday with the ScaleTransform removed, and maybe with another design when a mobile tab is selected.

**Comment by kikipoulet** [comment]
Solved in 4.5.5
